modules/update-service-create-service-web-console.adoc

```diff
@@ -0,0 +1,40 @@
+[id="update-service-create-service-web-console_{context}"]
+= Creating an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to create an Update Service application by using the OpenShift Update Service Operator.
```

Comment:
Update Service -> update service
Or Should it be "OpenShift Update Service application"?

Reply:
Went with the latter.